components/script/dom/window.rs

```diff
@@ -183,6 +184,8 @@ pub struct Window {
     history: MutNullableJS<History>,
     custom_element_registry: MutNullableJS<CustomElementRegistry>,
     performance: MutNullableJS<Performance>,
+    #[ignore_heap_size_of = "Arc"]
+    paint_time_metrics: Arc<PaintTimeMetrics>,
```

Comment:
At the moment we have a layout thread per window, and we don't expose this data through the DOM at all. I am inclined to keep it as simple as possible while we can.